frontend/src/pages/Home/components/JoinUs/JoinUs.jsx

```diff
@@ -15,13 +15,13 @@ export const JoinUs = (props) => {
             : `${styles["join-us"]} ${styles["join-us-light"]}`
         }
       >
-        <h1 className={styles["title"]}>Join Us</h1>
+        <h1 className={styles["title"]}>Why not join us?</h1>
```

Comment:
Why not join us? It is producing a negative impact 
You can change it with some other catchy line :)